jenkins/test/validators/yaml.sh

```diff
@@ -0,0 +1,24 @@
+base_sha=$PRV_BASE_SHA
+remote_sha=$PRV_REMOTE_SHA
+
+# Overwrite sha's with provided command-line arugments
+if [ -n $1 ]; then
+    base_sha=$1
+fi
+if [ -n $2 ]; then
+    remote_sha=$2
+fi
+
+# Download the latest yaml validation script
+curl https://raw.githubusercontent.com/openshift/openshift-ansible/master/git/yaml_validation.py -o jenkins/test/validators/yaml_validation.py 2>&1
```

Comment:
We recently changed this so that yaml_validation.py is in our repository instead of pulling from openshift-ansible.  We need to make sure that we are doing that new style instead of this as we found a bug.